docs/4____network_abstraction.adoc

```diff
@@ -314,9 +315,10 @@ Attribute definitions::
  * `name` is the network name, e.g., `Powertrain`, see <<low-cut-example, example>> and constraints.
 
 Element definitions::
- * Every Bus Terminal contains exactly four variables each in the role of: `Tx_Data`, `Tx_Clock`, `Rx_Data` or `Rx_Clock`.
+ * Every Bus Terminal contains at least four variables each in the role of: `Tx_Data`, `Tx_Clock`, `Rx_Data` or `Rx_Clock`.
  * Variables in the role of `Rx_Data`, `Tx_Data`, `Rx_Clock` and `Tx_Clock` are exposed in the `modelDescription.xml` and must be listed as `<TerminalMemberVariable>` elements, whereby `variableName` can be freely chosen.
  * `<TerminalMemberVariable>.memberName` defines the role of the variable within the `<Terminal>` and shall be set to `Tx_Data`, `Tx_Clock`, `Rx_Data` or `Rx_Clock`.
+* The `name` of a `<Terminal>` for additional configuration parameters is equal to `Configuration` and is nested within the actual Bus Terminal.
  * Multiple use of the same variable within different Bus Terminals is not allowed.
```

Comment:
Shouldn't it be possible to have one parameter/variable for configuration that can be used in the terminal for different busses?

Reply:
If we explicitly define the nested terminal "Configuration", this should be clearer anyway.
In different the "Configuration" terminals of the same fmu, variable can be reused.